examples/gadgets/withfilter/trace/network/README.md

```diff
@@ -0,0 +1,127 @@
+# trace network example (with filter)
+
+This example shows how to use different packages to create an application to
+trace network connections filtering by containers. Beyond that, network events
+are enriched with Kubernetes IP addresses and Kubernetes destinations, e.g. pod
+or service names.
+
+## How to build
+
+```bash
+$ go build .
+```
+
+## How to run in a terminal
+
+``` console
+$ sudo ./network
+failed to initialize kube IP enricher: creating k8s inventory cache: creating new k8s clientset: stat /root/.kube/config: no such file or directory. It won't be available.
+failed to initialize kube name enricher: creating k8s inventory cache: creating new k8s clientset: stat /root/.kube/config: no such file or directory. It won't be available.
+[...]
+{
+  "runtime": {
+    "runtimeName": "containerd",
+    "containerId": "e3bc6362dac572037f025f898b394c698c701e1dba753099459a1889b6ecd95b",
+    "containerName": "coredns",
+    "containerImageName": "registry.k8s.io/coredns/coredns:v1.10.1",
+    "containerImageDigest": "sha256:a0ead06651cf580044aeb0a0feba63591858fb2e43ade8c9dea45a6a89ae7e5e"
+  },
+  "k8s": {
+    "namespace": "kube-system",
+    "podName": "coredns-5dd5756b68-wh964",
+    "containerName": "coredns"
+  },
+  "timestamp": 1710410447833705750,
+  "type": "normal",
+  "mountnsid": 4026533116,
+  "netnsid": 4026532725,
+  "pid": 4955,
+  "tid": 5150,
+  "comm": "coredns",
+  "uid": 0,
+  "gid": 0,
+  "pktType": "OUTGOING",
+  "proto": "TCP",
+  "port": 8080,
+  "podHostIP": "172.16.184.153",
+  "podIP": "192.168.160.244",
+  "podOwner": "coredns",
+  "podLabels": {
+    "k8s-app": "kube-dns",
+    "pod-template-hash": "5dd5756b68"
+  },
+  "dst": {
+    "addr": "127.0.0.1",
+    "version": 4,
+    "kind": "raw"
+  }
+}
+[...]
+```
+
+## How to deploy the DaemonSet
+
```

Comment:
nit: add a comment indicating this work on a cluster with containerd, otherwise tell the user to build and push the images to their container registries by themselves?

Reply:
Done